src/components/Input/Input.spec.js

```diff
@@ -3,6 +3,24 @@ import React from 'react';
 import Input from '.';
 import Label from '../Label';
 
+const DummyIcon = () => (
```

Comment:
Is this really necessary? Can't you just use a div with the same dimensions? After all, you renamed the render props to `prefix` and `suffix` because we wouldn't use icons exclusively here.

Reply:
Good point. I'll simplify 👍